metabase/metabase
# Add metabot tracking

### Description

Adds standalone Metabot page tracking. Also, it mocks the feedback endpoint since we started tracking snowplow events on the FE and it is tested without BE involvement.

### How to verify

Run snowplow [mock server](https://www.notion.so/metabase/Snowplow-integration-5da1f874beda4153b4fccfa6c1e77caa#351e05126ce14fdd8e08576980fd4728). Ensure Metabot [events](https://docs.google.com/document/d/1kknBaY8ZoiFNsj3dI3d4Jo1xlD_s8Vq83PFXzi53NZI/edit#) are tracked:

### Checklist



- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/30281)
